Gifski/ConversionCompletedViewController.swift

```diff
@@ -23,6 +23,13 @@ final class ConversionCompletedViewController: NSViewController {
 		maxWidth: 260
 	)
 
+	private lazy var copyTooltip = Tooltip(
```

Comment:
ah yeah due to the single branch the changes from the other PR got here too 😢

Reply:
My bad. Will fix this.